code/modules/mob/living/carbon/human/life.dm

```diff
@@ -1393,13 +1393,6 @@ var/global/list/brutefireloss_overlays = list("1" = image("icon" = 'icons/mob/sc
 			if (getToxLoss() >= 45 && nutrition > 20)
 				vomit()
 
-		//0.1% chance of playing a scary sound to someone who's in complete darkness

```

Comment:
... this should probably be a prob(1) chance, but i don't think you should remove it